tools/mo/openvino/tools/mo/moc_frontend/pytorch_frontend_utils.py

```diff
@@ -39,18 +41,43 @@ def to_torch_tensor(tensor):
         return tensor
     if isinstance(tensor, np.ndarray):
         return torch.tensor(tensor)
-    if isinstance(tensor, np.ndarray):
-        return torch.tensor(tensor)
     if isinstance(tensor, Tensor):
         return torch.tensor(tensor.data)
     if isinstance(tensor, (float, int, bool)):
         return tensor
     else:
         raise Error("Unexpected type of example_input. Supported types torch.Tensor, np.array or ov.Tensor. "
                     "Got {}".format(type(tensor)))
-
-
-def prepare_torch_inputs(example_inputs, input_shape, allow_none=False):
+def get_torch_dtype(dtype):
+    import torch
+    ov_str_to_torch = {
+        "boolean": torch.bool,
+        "f16": torch.float16,
+        "f32": torch.float32,
+        "f64": torch.float64,
+        "i8": torch.int8,
+        "i16":torch.int16,
+        "i32": torch.int32,
+        "i64": torch.int64,
+        "u8": torch.uint8,
+    }
+    if dtype is None:
+        return torch.float
```

Comment:
Convenient enough? For example if an LM is being converted what kind of error I get if dtype is not specified?

Reply:
it depends from model and ability to script it (in case if scripting works, we just ignore example inputs), if not for bert it will be: 
RuntimeError: Expected tensor for argument #1 'indices' to have one of the following scalar types: Long, Int; but got torch.FloatTensor instead (while checking arguments for embedding)

Do you mean we should provide some warning that the default type selected is torch.float32, please specify directly if different type required?